<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displaywidget</class>
 <widget class="QDockWidget" name="displaywidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>306</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="iterationsLabel">
            <property name="text">
             <string>Iterations:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="iterationsSlider">
            <property name="minimum">
             <number>11</number>
            </property>
            <property name="maximum">
             <number>2400</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>400</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
            <property name="tickInterval">
             <number>200</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="strategyLabel">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Strategy:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="strategyCombo">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="DisplayGLWidget" name="glDisplayWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>OpenHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayGLWidget</class>
   <extends>QWidget</extends>
   <header>displayglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
